src/nspawn/nspawn-settings.h

```diff
@@ -56,9 +56,19 @@ typedef enum SettingsMask {
         SETTING_CPU_AFFINITY      = UINT64_C(1) << 20,
         SETTING_RLIMIT_FIRST      = UINT64_C(1) << 21, /* we define one bit per resource limit here */
         SETTING_RLIMIT_LAST       = UINT64_C(1) << (21 + _RLIMIT_MAX - 1),
-        _SETTINGS_MASK_ALL        = (UINT64_C(1) << (21 + _RLIMIT_MAX)) - 1
+        _SETTINGS_MASK_ALL        = (UINT64_C(1) << (21 + _RLIMIT_MAX)) - 1,
+        _FORCE_ENUM_WIDTH         = UINT64_C(-1)
```

Comment:
hmm, wouldn't it be nicer to define this as UINT64_MAX instead? I think that would be a lot more readable as defining a literal negative value as unsigned always strikes me very weird...